### PR TITLE
ticket 8 is completed

### PR DESCRIPTION
**Pull request by yazyounas**

### CODE REVIEW

File: src/Components/home_components/PostComment.jsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            name="author"
+            value={author}
+            onChange={(event) => setAuthor(event.target.value)}
+            required
+          />
+        </div>
+        <button type="submit" className="submit-button">Post Comment</button>
+      </form>
+    </main>
+  );
```
Comment (Poonam-raj) [inline src/Components/home_components/PostComment.jsx:79]: lovely controlled components!

File: src/Components/home_components/PostComment.jsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      <form onSubmit={handleSubmit} className="comment-form">
+        <div className="form-group">
+          <label htmlFor="body">Comment:</label>
+          <textarea
+            id="body"
+            name="body"
+            value={body}
+            onChange={(event) => setBody(event.target.value)}
+            required
+          />
```
Comment (Poonam-raj) [inline src/Components/home_components/PostComment.jsx:57]: good choice of textarea for the comment input

**Review by Poonam-raj — CHANGES_REQUESTED** [review]
Hey Yasir here is my feedback:

1. It is a slightly strange user experience to have to type in my username, especially when you have a log in button at the top of the app. It would be better at this point to have a hardcoded logged in user and not giving the user the option of typing in the username. This can be accompanied by the log in button indicating who they are logged in as.
 
2. Very nice feedback to the user about failures and success 🎉  and immediate feedback with the new comment appearing at the top - very nice

3. You need to stop a user from posting multiple times when the internet is slow. I throttled my internet on the network tab of my browser so it acted slow and clicked the submit button loads of times and it let me post my comment lots of times. We want to prevent this from happening by disabling the button or the form in some way when the post is in progress.

^^^ the two points I've got here please address them and resubmit a PR - nice work so far!

**Review by callumu21 — CHANGES_REQUESTED** [review]
Nice job preventing a user from being able to make multiple submission requests while the first is still being processed! As for removing the log in functionality, you seem to have moved in the opposite direction based on Poonam's feedback as we'll need a logged in user when it comes to deletions later on, so having it in place here makes more sense than requiring someone to enter their username for every comment post.

Can you either bring back the log-in functionality, or hardcode a logged-in user in state/context. The username value when posting should come from that user state rather than it being provided by the user.

Once you've done that, resubmit this PR and we can take another look 👍

**Pull request merged**